custom/enikshay/templates/enikshay/partials/report_content.html

```diff
@@ -0,0 +1,63 @@
+{% load i18n %}
```

Comment:
Currently, I don't have better idea than copy-paste those part of code from corehq